File: tools/sms/src/main/java/top/bettercode/summer/tools/sms/aliyun/AliSmsSignResponse.java
package top.bettercode.summer.tools.sms.aliyun;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;

public class AliSmsSignResponse extends AliSmsResponse {

  @JsonProperty("SmsSignList")
  private List<SmsSignListItem> smsSignList;

  public void setSmsSignList(List<SmsSignListItem> smsSignList) {
    this.smsSignList = smsSignList;
  }

  public List<SmsSignListItem> getSmsSignList() {
    return smsSignList;
  }

  @Override
  public String toString() {
    return "AliSmsSignResp{" +
        "smsSignList=" + smsSignList +
        "} " + super.toString();
  }
}

File: clients/sms/src/main/kotlin/top/bettercode/summer/tools/sms/aliyun/AliSmsSignResponse.kt
package top.bettercode.summer.tools.sms.aliyun

import com.fasterxml.jackson.annotation.JsonProperty

class AliSmsSignResponse : AliSmsResponse() {
    @JsonProperty("SmsSignList")
    var smsSignList: List<SmsSignListItem>? = null
    override fun toString(): String {
        return "AliSmsSignResp{" +
                "smsSignList=" + smsSignList +
                "} " + super.toString()
    }
}